src/mongo/db/storage/wiredtiger/wiredtiger_record_store_test.cpp

```diff
@@ -1449,71 +1453,27 @@ TEST(WiredTigerRecordStoreTest, OplogStones_ReclaimStones) {
 
         wtrs->reclaimOplog(opCtx.get());
 
-        ASSERT_EQ(3, rs->numRecords(opCtx.get()));
-        ASSERT_EQ(320, rs->dataSize(opCtx.get()));
-        ASSERT_EQ(2U, oplogStones->numStones());
+        ASSERT_EQ(2, rs->numRecords(opCtx.get()));
+        ASSERT_EQ(190, rs->dataSize(opCtx.get()));
+        ASSERT_EQ(1U, oplogStones->numStones());
         ASSERT_EQ(1, oplogStones->currentRecords());
         ASSERT_EQ(50, oplogStones->currentBytes());
     }
 
-    // No-op if the number of oplog stones is less than or equal to the number of stones to keep.
+    // No-op if dataSize <= cappedMaxSize.
     {
         ServiceContext::UniqueOperationContext opCtx(harnessHelper.newOperationContext());
 
         wtrs->reclaimOplog(opCtx.get());
 
-        ASSERT_EQ(3, rs->numRecords(opCtx.get()));
-        ASSERT_EQ(320, rs->dataSize(opCtx.get()));
-        ASSERT_EQ(2U, oplogStones->numStones());
+        ASSERT_EQ(2, rs->numRecords(opCtx.get()));
+        ASSERT_EQ(190, rs->dataSize(opCtx.get()));
+        ASSERT_EQ(1U, oplogStones->numStones());
         ASSERT_EQ(1, oplogStones->currentRecords());
         ASSERT_EQ(50, oplogStones->currentBytes());
     }
 }
 
-// Verify that oplog stones are not reclaimed even if the size of the record store exceeds
-// 'cappedMaxSize'.
-TEST(WiredTigerRecordStoreTest, OplogStones_ExceedCappedMaxSize) {
```

Comment:
because in the original implementation, there are two benchmarks: oplogStoneNum  and cappedMaxSize.  In my 
new implementation, only cappedMaxSize is considered,  the oplogStoneNum is no longer used. I have made TEST(WiredTigerRecordStoreTest, OplogStones_ReclaimStones) passed, which in the new implementation is duplicated with TEST(WiredTigerRecordStoreTest, OplogStones_ReclaimStones)